<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEditDisplaySheet">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>601</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="graphicsViewChart">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>601</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxFunctions">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>221</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Eliga una funci√≥n</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QStackedWidget" name="stackedWidgetDiagramFunctions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>201</width>
       <height>431</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSelectFunction">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonGenerateDiagram">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Generar Diagrama</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonExportDiagram">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Exportar Diagrama</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Administrar archivos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="buttonImportDocuments">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Importar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="menuImportDocuments"/>
    <addaction name="separator"/>
    <addaction name="menuQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuImportDocuments">
   <property name="text">
    <string>Importar</string>
   </property>
  </action>
  <action name="menuQuit">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
